src/core/inline-idl-parser.js

```diff
@@ -29,35 +29,35 @@ function parseInlineIDL(str) {
     if (methodRegex.test(value)) {
       const [, identifier, allArgs] = value.match(methodRegex);
       const args = allArgs.split(/,\s*/).filter(arg => arg);
-      results.push({ type: "method", identifier, args });
+      results.push({ type: "method", identifier, args, str });
```

Comment:
Could we pass the result of `str.includes("/")` instead, with its field name something like `explicitContext`?

Reply:
or `renderParent`